### ISSUE TITLE
Why might the openAll method not work?

### ISSUE THREAD

**Issue by zetorby** [minop1205/react-dnd-treeview#126]
Why might the openAll method not work?
Instead of opening all nodes, it closes them....

**Comment by minop1205** [comment]
@zetorby Thanks for the report.

Can you tell me what version of the package you are using?
Also, any code, etc. to reproduce the problem would be helpful.

Also, is CodeSandbox here working correctly?

https://codesandbox.io/s/opening-and-closing-all-nodes-js-eqxzti

**Comment by zetorby** [comment]
@minop1205 Hi.
I think the problem is in the ID
I changed the IDs of elements in your example, and now the method doesn't work correctly
https://codesandbox.io/s/opening-and-closing-all-nodes-js-forked-8cokxt?file=/src/sample_data.json

**Comment by minop1205** [comment]
@zetorby Thanks for the sample.

I have looked at the sample you provided and it appears that both `openAll` and `closeAll` are working as expected.

The current implementation of `openAll` is to open all nodes where `droppable` is set to `true`.

Therefore, the following node in `sample_data.json` has `droppable` set to `false`, so the nodes under it are not displayed.

```
{
    "id": 277,
    "parent": 276,
    "droppable": false,
    "text": "File 1-1",
    "data": {
      "fileType": "csv",
      "fileSize": "0.5MB"
    }
  },
```

I think if I set `droppable` on this node to `true`, it will work as expected, has this solved the problem?
